components/map/index.js

```diff
@@ -1,17 +1,25 @@
 /* eslint-disable react/no-unused-state */
 import React, { Component } from 'react';
-import MapView, { Polygon, PROVIDER_GOOGLE } from 'react-native-maps';
+import MapView, { Polyline, Polygon, PROVIDER_GOOGLE } from 'react-native-maps';
+import { View } from 'react-native';
 import buildings from '../../assets/polygons/polygons';
 import styles from './styles';
 
 export default class TheMap extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      region: {
+      coordinate: {
```

Comment:
I don't see this variable being used anywhere, was this intended for future use?